src/javaRosa.js

```diff
@@ -1117,9 +1117,7 @@ define([
 
         var items = getItextItemsFromMugs(form, true);
         items = _.object(_.map(items, function (item) { return [item.id, item]; }));
-        cells = nextRow();
-        while (cells) {
-            // what's the point of creating items here?
+        for (cells = nextRow(); cells; cells = nextRow()) {
```

Comment:
Interesting comment, @czue. I do not believe I am using that feature here, am I? A more concise way to write this is
```js
while (cells = nextRow()) {
```
but our lint tool doesn't like it (and I think that statement looks pretty misleading), so I opted for the `for` loop instead.